src/mpi/spawn/comm_join.c

```diff
@@ -180,7 +180,7 @@ int MPI_Comm_join(int fd, MPI_Comm *intercomm)
 
     mpi_errno = MPIR_Close_port_impl(local_port);
     if (mpi_errno) MPIR_ERR_POP(mpi_errno);
-
+    MPIR_ERRTEST_ARGNULL(intercomm, "intercomm", mpi_errno);
```

Comment:
This also removes a line of whitespace. We should try not to do that when doing a simple addition, such as this.